git/homework.py

```diff
@@ -116,18 +113,23 @@ def remove_from_list_all_negative_numbers(data: List[int]) -> list:
         remove_from_list_all_negative_numbers([1, 5, -7, 8, -1])
         >>> [1, 5, 8]
     """
-    pass
+    return [x for x in data if x >= 0]
 
 
 def alphabet() -> dict:
     """
-    Create dict which keys is alphabetic characters. And values their number in alphabet
+    Create dict which keys is numbers staring from 1. And values are alphabetic characters corresponding to the number.
     Notes You could see an implementaion of this one in test, but create another one
     Examples:
         alphabet()
-        >>> {"a": 1, "b": 2 ...}
+        >>> {"1": a, "2": b ...}
     """
-    pass
+    alph = {}
+    n = 1
+    for i in range(ord('a'), ord('z') + 1):
+        alph[n] = chr(i)
+        n += 1
+    return alph 
```

Comment:
Recommendation: read about [Dict Comprehension](https://www.datacamp.com/community/tutorials/python-dictionary-comprehension). 
Your code can be made more beautiful using dict comprehesion:
` {key: chr(value) for key, value in enumerate(range(97, 123), 1)}
`